contracts/base/Dispatcher.sol

```diff
@@ -356,6 +351,19 @@ abstract contract Dispatcher is Payments, V2SwapRouter, V3SwapRouter, Callbacks,
                 }
                 bytes calldata data = inputs.toBytes(1);
                 (success, output) = SEAPORT_V1_4.call{value: value}(data);
+            } else if (command == Commands.LOOKS_RARE_V2) {
```

Comment:
Is there a point in keeping `Commands.LOOKS_RARE_721` and `Command.LOOKS_RARE_1155` around?? maybe we're afraid they might have an emergency rollback?